examples/grpc_proxying/invoke-caller.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+import logging
+
+import grpc
+import helloworld_service_pb2_grpc
+from helloworld_service_pb2 import HelloRequest, HelloReply
+from dapr.clients import DaprClient
+import json, time
+
+async def run() -> None:
+    async with grpc.aio.insecure_channel('127.0.0.1:50007') as channel:
+        metadata = (('dapr-app-id', 'server'),)
```

Comment:
Will this be necessary for every call that the user makes to their gRPC service? Is it possible to provide this automatically? Perhaps when you construct the client?

Reply:
So the idea here is to follow gRPC proxying: [You can see here](https://docs.dapr.io/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc/)
This way a client can just use their grpc client to call the Dapr sidecar without using the client directly and Dapr will redirect it to the right service. All only by adding metadata.